examples/ibm-cloudant-database/modules/config-database/README.md

```diff
@@ -0,0 +1,30 @@
+# Module Cloudant Database
+
+This module is used to create cloudant database.
+
+## Example Usage
+```
+module "cloudant-database" {
+  //Uncomment the following line to point the source to registry level
+  //source = "terraform-ibm-modules/cloudant/ibm//modules/database"
+
+  source                         = "../../modules/database"
+  db_name                        = var.db_name
+  cloudant_instance_crn          = var.instance_crn
+  cloudant_database_partitioned  = var.is_partitioned
+}
```

Comment:
I assume we are intentionally omitting `cloudant_database_q ` here because dragons?

Reply:
Yeah, let's not encourage people to use it too much.